applications/zpc/components/zwave_command_classes/src/zwave_command_class_user_code.c

```diff
@@ -875,6 +878,19 @@ static void zwave_command_class_user_code_on_number_of_users_update(
   attribute_store_node_t endpoint_node
     = attribute_store_get_first_parent_with_type(updated_node,
                                                  ATTRIBUTE_ENDPOINT_ID);
+  attribute_store_node_t version_node
+    = attribute_store_get_first_child_by_type(endpoint_node,
+                                              ATTRIBUTE(VERSION));
+  // We need to check the version of the supporting node:
+  zwave_cc_version_t supporting_node_version = 0;
+  attribute_store_get_reported(version_node,
+                               &supporting_node_version,
+                               sizeof(supporting_node_version));
+  if (supporting_node_version == 1) {
+    sl_log_debug(LOG_TAG, "Skip creating User ID nodes for CC UserCode V1.");
+    return;
+  }
+
```

Comment:
Is there a reason behind this change ? I think CC UserCode V1 support multiple user ids.

Reply:
As described above, we do not want to create all 255 Used IDs during Interview phase and wait 5 minutes for it to complete.